src/ontology/cl-edit.owl

```diff
@@ -32378,6 +32381,30 @@ SubClassOf(obo:CL_4042022 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_000101
 SubClassOf(obo:CL_4042022 ObjectSomeValuesFrom(obo:RO_0002203 obo:CL_0000127))
 SubClassOf(obo:CL_4042022 ObjectSomeValuesFrom(obo:RO_0002292 obo:PR_000001307))
 
+# Class: obo:CL_4042023 (pthlh-expressing interneuron)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:30134177") Annotation(oboInOwl:hasDbXref "PMID:37292997") obo:IAO_0000115 obo:CL_4042023 "A GABAergic interneuron that has its soma in a striatum. This GABAergic interneuron is characterised by the expression of PTHLH and PVALB.")
```

Comment:
This reads like it covers all GABAergic neurons in the striatum.  

proposed change to 

A PTHLH expressing GABAergic neuron with its soma in the striatum.  

IS PVALB really expressed in all?
![image](https://github.com/user-attachments/assets/6635b706-7038-497e-896a-e816b1f3cb3b). Maybe mention spatial gradient of PVALB across these cells in the gloss?